### PR TITLE
Update to add Schema headings, other minor fixes.

### PR DESCRIPTION
**Pull request by RoAnnC1**

### CODE REVIEW

File: src/api-reference/travel-profile/v1.notification-company-resource.markdown [hunk 1]
```diff
@@ -188,29 +190,29 @@ Subscribes or unsubscribes the partner from notifications when the company's Tra
...
-type=profile|Required. The type of subscription.
+`type=profile`|Required. The type of subscription.
 
 **Examples**
 
 To subscribe:
 
 ```
-https://www.concursolutions.com/api/company/v1.0/subscribe?type=profile
+https://www.SAP Concursolutions.com/api/company/v1.0/subscribe?type=profile
```
Comment (jonikamelcher) [inline src/api-reference/travel-profile/v1.notification-company-resource.markdown]: Find and replace went trigger happy.

File: src/api-reference/travel-profile/v1.notification-company-resource.markdown [hunk 2]
```diff
@@ -188,29 +190,29 @@ Subscribes or unsubscribes the partner from notifications when the company's Tra
...
 ```
 
 To unsubscribe:
 
 ```
-https://www.concursolutions.com/api/company/v1.0/unsubscribe?type=profile
+https://www.SAP Concursolutions.com/api/company/v1.0/unsubscribe?type=profile
```
Comment (jonikamelcher) [inline src/api-reference/travel-profile/v1.notification-company-resource.markdown]: Ditto.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 3]
```diff
@@ -360,49 +367,49 @@ Where `access_token` is the OAuth 2.0 access token of the user whose itinerary i
 
 application/xml
 
-## Response
+## <a name="gettd-schema"></a>Get Trip Details Response Schema
```
Comment (jonikamelcher) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown:371]: This is getts-schema in the TOC. Which is correct?

File: src/api-reference/travel/itinerary/booking/booking-resource.markdown [hunk 4]
```diff
@@ -47,57 +50,60 @@ Examples:
...
+|  `TicketPickupNumber` |  The confirmation number to pick up the booked ticket, if available. |
+|  `AirfareQuotes` |  List of stored airfare quotes for this booking. |
+|  `AirlineTickets` |  List of Airline Tickets for this booking. |
+|  `Charges` |  List of Charges for this booking. |
+|  `MiscChargeOrders` |  List of Miscellaneous `AirCharges` for this booking. |
+|  `Passengers` | The `Passengers` element contains a `Passenger` child element for each booked passenger. The description of each child element can be seen in a subsequent table. |
+|  `PassPrograms` |  List of Pass Programs for this booking. |
+|  `PhoneNumbers` |  List of Phone numbers associated with this booking. |
+|  `RailPayments` |  List of Rail payments associated with rail segments in this booking. |
+|  `Segments` |  List of Segments in this booking. This parent element contains one or more `Air, `Car`, `Hotel`, `Dining`, `Ride`, `Rail`, `Parking`, or `Event` parent elements for the booking. Refer to [Booking Object Elements][1] for more information about the child elements contained in the booking elements. |
```
Comment (jonikamelcher) [inline src/api-reference/travel/itinerary/booking/booking-resource.markdown]: Missing a backtick around Air.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 5]
```diff
@@ -52,19 +56,19 @@ To identify a specific user by login ID or `XMLSyncID`, you can specify the foll
...
+|`ItemsPerPage`|`number`|`integer`|The `includeMetadata` query parameter combined with the `ItemsPerPage` and `Page` query parameters will cause the response to be divided into pages. The response will be wrapped in a `ConcurResponse` parent element, with both the response details and the paging metadata included. If the `ItemsPerPage` query parameter is not sent, the response will default to 200 if the `Page` query parameter is sent, or 1000 if the `Page` query parameter is not set. If the `Page` query parameter is not sent, the response will default to page 1.|
+|`includeVirtualTrip`|`flag`|`integer`	|Virtual trips are segments booked offline through the Concur Request product. Set the `includeVirtualTrip` query parameter to 1 to include those trips in the list.|
+|includ`eCanceledTrips	|`true/false`|`string`|The `includeCanceledTrips` query parameter will cause the request to also return trips with a status of Canceled. When this query parameter is set to true, the response will include the `TripStatus` element.|
```
Comment (jonikamelcher) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Rogue backtick.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 6]
```diff
@@ -360,49 +367,49 @@ Where `access_token` is the OAuth 2.0 access token of the user whose itinerary i
...
-|BookingSource	|string	|The name of the booking source for this booking. A booking source is a textual name the system uses to track where a booking took place.
-|DateModifiedUtc	|dateTime	|The date the booking was last modified, in UTC. Format: YYYY-MM-DDThh:mm:ss.
-|DateBookedLocal	|dateTime	|The date the booking was created, in the booking location's local time. Format: YYYY-MM-DDThh:mm:ss.
-|ItinSourceName	|string	|The itinerary source. Format: TravelSupplier.
-|PassengerCount	|integer	|The number of passengers included in the booking.
+|`Segments`	|`array`	|List of segments in this booking. The child elements included in this element vary depending on whether a TMC, SAP Concur client,  third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs, clients, and third-party developers**, the `Segments` element contains one or more `Air`, `Car`, `Hotel`, Di`ning, `Ride`, `Rail`, `Parking`, or `Travel` parent elements. **For TripLink suppliers**, the `Segments` element contains one or more `Air`, `Car`, `Hotel`, or `Ride` parent elements.
```
Comment (jonikamelcher) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Dining has a rogue backtick that is throwing everything off.

File: src/api-reference/user/index.markdown [hunk 7]
```diff
@@ -27,7 +35,7 @@ Name|Type|Format|Description
 -----|------|------|--------------
 `loginID`|`string`|-|The URL-encoded SAP Concur login of the user. Optional.
 
-### Response
+### <a name="getUserSchema"></a>Get User Response Schema
```
Comment (jonikamelcher) [inline src/api-reference/user/index.markdown:39]: I know you didn't make any changes to this but there is some stuff that is just straight up inaccurate here. Could you please remove the Format from the end of the description into the appropriate Format column? Also, if the format is a Y/N, there's no way the type is a string, please change those to boolean.

File: src/api-reference/travel-profile/v1.loyalty-program-resource.markdown [hunk 8]
```diff
@@ -47,40 +49,38 @@ If the request is sent by a TMC, the request can update any loyalty program for
...
+#### Accept Header
 
 application/xml
 
-### Request body
-
-#### Root elements
+### Request Body
 
 This request contains the **LoyaltyMembershipUpdate** parent element with a **Membership** child element for each included loyalty program. The **Membership** element has a **UniqueID** attribute containing the loyalty program identifier, and the following child elements.
```
Comment (jonikamelcher) [inline src/api-reference/travel-profile/v1.loyalty-program-resource.markdown]: Can you throw these fields into backticks? I wouldn't normally mentioned it but you did it for all the other examples of this in the doc set.

File: src/api-reference/travel-profile/v1.loyalty-program-resource.markdown [hunk 9]
```diff
@@ -156,14 +156,14 @@ Authorization: OAuth {access token}
 
 application/xml
 
-###  Response body root elements
+### <a name="responseschema"></a>Update Loyalty Program Membership Response Schema
 
 This request will return a **LoyaltyMembershipResponse** parent element with the following child elements:
```
Comment (jonikamelcher) [inline src/api-reference/travel-profile/v1.loyalty-program-resource.markdown]: Backticks.

**Review by jonikamelcher — CHANGES_REQUESTED** [review]

File: src/api-reference/travel-profile/v1.notification-company-resource.markdown (hunk 2 above)
Comment (RoAnnC1) [reply src/api-reference/travel-profile/v1.notification-company-resource.markdown]: Fixed

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 3 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown:371]: Actually, I didn't add a schema entry for this heading. getts-schema was the previous one.
 Fixed.

File: src/api-reference/travel-profile/v1.notification-company-resource.markdown (hunk 1 above)
Comment (RoAnnC1) [reply src/api-reference/travel-profile/v1.notification-company-resource.markdown]: Fixed

File: src/api-reference/travel/itinerary/booking/booking-resource.markdown (hunk 4 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/booking/booking-resource.markdown]: Fixed

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 5 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Fixed

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 6 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Fixed

File: src/api-reference/travel-profile/v1.loyalty-program-resource.markdown (hunk 8 above)
Comment (RoAnnC1) [reply src/api-reference/travel-profile/v1.loyalty-program-resource.markdown]: Fixed

File: src/api-reference/travel-profile/v1.loyalty-program-resource.markdown (hunk 9 above)
Comment (RoAnnC1) [reply src/api-reference/travel-profile/v1.loyalty-program-resource.markdown]: Fixed

File: src/api-reference/user/index.markdown (hunk 7 above)
Comment (RoAnnC1) [reply src/api-reference/user/index.markdown:39]: Fixed

**Review by jonikamelcher — APPROVED** [review]

**Pull request merged**